<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ds2423ui</class>
 <widget class="QWidget" name="ds2423ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Icons</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="RomID">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="scratchpad">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QSpinBox" name="Offset">
     <property name="prefix">
      <string>Delta Max : </string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_scratchpad_4">
     <property name="text">
      <string>Delta</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="Delta">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelRomID">
     <property name="text">
      <string>RomID : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_scratchpad_3">
     <property name="text">
      <string>Compteur</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QComboBox" name="counterMode"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="Compteur">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="countInit">
     <property name="title">
      <string>Offset Ajdust</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="Interval"/>
      </item>
      <item row="1" column="0">
       <widget class="QDateTimeEdit" name="nextOffsetAdjust">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="SaveOnUpdate">
        <property name="text">
         <string>Save only on update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_scratchpad_2">
     <property name="text">
      <string>Scrathcpad : </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
